Dockerfile

```diff
@@ -47,17 +46,15 @@ ENV JAVA_HOME /usr/lib/jvm/java-8-oracle
 # Install python 2.7 and dependancies for Genetorrent.
 #
 
-RUN apt-get install -y python && \
-    wget -qO- https://pypi.python.org/packages/source/P/PyYAML/PyYAML-3.11.tar.gz#md5=f50e08ef0fe55178479d3a618efe21db | \
-    tar xvz --strip-components 1 && \
-    python setup.py install && \
-    mkdir -p /icgc/cli/clients
+RUN apt-get install -y python-pip && \
+    pip install -U pip setuptools
 
-COPY /lib/cli/ /icgc/cli/
+COPY . /icgc/icgcget/
+COPY /conf/ /icgc/conf
 
-RUN cd /icgc
-
-ENV PATH=$PATH:/icgc/cli
+RUN cd /icgc/icgcget && \
```

Comment:
The comment above this is no longer accurate:

```
#
# Install python 2.7 and dependancies for Genetorrent.
#
```

Please update this to reflect that this is for `icgc-get`